node/src/accountant/mod.rs

```diff
@@ -329,40 +337,14 @@ impl Handler<ReportTransactionReceipts> for Accountant {
     }
 }
 
-#[derive(Debug, PartialEq, Eq, Message, Clone)]
-pub struct CancelFailedPendingTransaction {
-    pub id: PendingPayableId,
-}
-
-impl Handler<CancelFailedPendingTransaction> for Accountant {
+impl Handler<NewPendingPayableFingerprints> for Accountant {
```

Comment:
Hi. Other comments are okay. I just don't understand this one, sorry. 

Oh, but it might be a misunderstanding... because the CancelFailedTransaction message is not only moved but it's gone completely as well as `ConfirmPendingTransaction` (their use wasn't necessarily important and so I happily took them out)... I doubt I could make the PR clearer to read by moving it.

It think it doesn't matter now.

Reply:
No, I misread the differences. I incorrectly thought I saw the other Handlers were still there and the diff viewer was showing more differences than there would have been if the `Handler<NewPendingPayableFingerprints>` impl was added after the other handlers. Since you deleted them, they're not there to put it after. Disregard.